### PR TITLE
I18N-392 - Add pagination in Branch page to show Branch with large te…

### PR DESCRIPTION
**Pull request by DarKhaos**
…xt unit count

Added a new BranchTextUnitStatistics handler method and modified frontend components

### CODE REVIEW

File: webapp/src/main/resources/public/js/components/branches/BranchesSearchResults.js [hunk 1]
```diff
@@ -66,18 +72,46 @@ class BranchesSearchResults extends React.Component {
...
 
+    renderBranchTextUnitsPagination(id) {
+        return (
+            <Row>
+                <AltContainer store={BranchTextUnitsPaginatorStore}>
+                    <Paginator
+                        onPreviousPageClicked={() => {
+                            BranchesPageActions.changeSelectedBranchTextUnitIds([]);
+                            BranchTextUnitsPaginatorActions.goToPreviousPage();
+                            BranchTextUnitsParamActions.changeBranchStatisticId(id);
```
Comment (byronantak) [inline webapp/src/main/resources/public/js/components/branches/BranchesSearchResults.js:83]: I might be misunderstanding the full context but this seems odd to me.

Is this necessary? If the user is clicking back, does that not mean the id should not change?
You also set the id when the next is clicked as well. This leads me to think that maybe the id shouldn't change in these steps

File: webapp/src/main/resources/public/js/sdk/entity/BranchStatisticsContent.js [hunk 2]
```diff
@@ -39,7 +45,7 @@ export default class BranchStatisticsContent {
          *
          * @type {boolean}
          */
-        this.isTruncated = false;
+        this.paginated = false;
```
Comment (byronantak) [inline webapp/src/main/resources/public/js/sdk/entity/BranchStatisticsContent.js:48]: Nit: I prefer the scheme of `isX` for booleans. So I think `isPaginated` would be a better name

Comment (DarKhaos) [reply webapp/src/main/resources/public/js/sdk/entity/BranchStatisticsContent.js]: Done

File: webapp/src/main/resources/public/js/components/branches/BranchesSearchResults.js (hunk 1 above)
Comment (DarKhaos) [reply webapp/src/main/resources/public/js/components/branches/BranchesSearchResults.js]: I have removed those lines

**Review by maallen — APPROVED** [review]

**Pull request merged**